.github/Support_question.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Support Question
+about: Questions about using Symfony and its components
+
+---
+
+<!--
+    The Symfony Code of Conduct applies to all the activity on this repository.
+    See https://symfony.com/doc/current/contributing/code_of_conduct/index.html
+-->
+
+We use GitHub issues only to discuss about Symfony bugs and new features. For
+this kind of questions about using Symfony or third-party bundles, please use
+any of the support alternatives shown in https://symfony.com/support
```

Comment:
this is also pointing to symfony proper

i would find it ok keep people opening support questions in github - we had it in the past and it was much less volume than what symfony itself would have.

wdyt of adding some guideline similar to bug report to make people explain what they want to do and what the question is?